jira/resources.py

```diff
@@ -290,6 +291,20 @@ def find(
         else:
             path = self._resource.format(id)
         url = self._get_url(path)
+        self._find_by_url(url, params)
+
+    def _find_by_url(
+        self,
+        url: str,
+        params: Optional[Dict[str, str]] = None,
+    ):
+        """Finds a resource on the specified url.
```

Comment:
Could you elaborate in this docstring, as there is no return value, but it instead loads the url.
Perhaps the name itself can be changed.

I'm also not sure what `.find()`

Reply:
Sure. The description is just a copy of the original _find(), I had to split it on two: _find() -> _find_by_url() in order to just overload one in the IssueProperty resource class. The motivation is that, despite other resources, IssueProperty JSON description does not have a "self" keyword and that has to be set by the passed in URL of the request.

Reply:
Cool, that is interesting. 

I'm a bit cautious of the additional request that needs to be made for this edge case. But tests are happy so probably will be alright, I'll give it a think but should be good